xbmc/pvr/addons/PVRClients.h

```diff
@@ -64,9 +64,9 @@ namespace PVR
   };
 
   class CPVRClients : public ADDON::IAddonMgrCallback,
-                      public Observer,
-                      private CThread
+                      public Observer
   {
+  friend class CPVRClient;
```

Comment:
... and you prefer a <code>friend</code> to be able to call one single method over making that method <code>public</code>?